config.default

```diff
@@ -157,6 +157,14 @@ mpasInterpolationMethod = bilinear
 # computing the MOC.
 yearsPerCacheFile = 1
 
+# should remapping be performed with ncremap or with the Remapper class
+# directly in MPAS-Analysis
+useNcremap = False
+
+# The minimum weight of a denstination cell after remapping, below
+# which it is masked out.
+renormalizationThreshold = 0.01
```

Comment:
@xylar: can you please explain this more? Also, is there a typo somewhere in the sentence before `renormalizationThreshold = 0.01`?